src/pluto/rgb_image.cr

```diff
@@ -0,0 +1,69 @@
+require "./image"
+require "./bindings/*"
+require "./format/*"
+require "./operation/*"
+
+class Pluto::RGBImage < Pluto::Image
```

Comment:
Sorry I suggested an incorrect name, but I think it should be `RGBAImage`?

Reply:
I'm fine with that too. I don't think people would have been that confused without the `A` since there's already an `alpha` method :)